frameworklauncher/src/main/java/com/microsoft/frameworklauncher/common/model/LauncherConfiguration.java

```diff
@@ -55,6 +55,7 @@
   // is also limited to 100MB = 500K * 200 bytes/task.
   private Integer maxTotalTaskNumber = 500000;
 
+  private Integer amDefaultContainerBasePort = 2000;
   // ApplicationMaster Setup
```

Comment:
Move it between line 93 and 94.
"Default" can be removed.
Refine it as:

Range amContainerDynamicPortRange

Reply:
And comment it

---
In reply to: [167153721](https://github.com/Microsoft/pai/pull/318#discussion_r167153721) [](ancestors = 167153721)